cirq-google/cirq_google/transformers/__init__.py

```diff
@@ -12,6 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+"""Transformers for compiling to Google-specific Sycamore gate."""
```

Comment:
Optional: I'd say "Google-specific gates" since we may have others in future (and we're unlikely to remember to come back to fix the comment).